### DISCUSSION TITLE
I make a simple one-click package and I think you can share it

### DISCUSSION THREAD

**Discussion by dryade36513** [Panchovix/stable-diffusion-webui-reForge#85] [General]
![225013](https://github.com/user-attachments/assets/d215a1f9-1907-4ee2-afe2-9a2f3ac6423d)
I copied Forge's method to make a one-click installer, which also allows you to click update to update the latest code you uploaded, and then click run to run your webui-user.bat.

the pack you can download here:
[download](https://www.mediafire.com/file/nvr3r7w78yv4elw/Stable_Diffusion_reForge.zip/file)

**Comment by MemeticGitHubUser** [comment]
Sweet :)
